### PR TITLE
Release v1.2.0

### PR DESCRIPTION
**Pull request by mprahl**
* Upgrade Python packages in case the requirements have changed during package upgrades
* Split test dependencies since there is no need to install the test dependencies on a production system
* Upgrade dependencies 
* Bump version to v1.2.0

### CODE REVIEW

**Comment by mprahl** [comment]
@thatarchguy I'll fix the tests tonight. I forgot to modify the travis file.

I was thinking that the test-requirements probably don't need to have versions pinned, so I could delete the `test-requirements.txt` file and rename `test-requirements.in` to `test-requirements.txt`.

What are your thoughts?

**Comment by thatarchguy** [comment]
>I was thinking that the test-requirements probably don't need to have versions pinned, so I could delete the test-requirements.txt file and rename test-requirements.in to test-requirements.txt.

If we are splitting off the test requirements, then yeah pop it off unpinned to a .txt file for simplicity. I don't believe we should be compiling our test requirements separately. 

Another pattern I've seen that I don't know if I'm fond of yet is 
```
requirements/
    ./common.txt
    ./dev.txt
```

with `./dev` containing `-r common.txt` so you don't need multiple pip install declarations.

**Comment by mprahl** [comment]
@thatarchguy I had to make some changes based on some breaking changes from `passlib` and a deprecation warning from `flask-wtf`, but it should be good to go now.

File: postmaster/forms.py [hunk 1]
```diff
@@ -4,14 +4,14 @@
 Purpose: form definitions for the app
 """
 
-from flask_wtf import Form
+from flask_wtf import FlaskForm
```
Comment (thatarchguy) [inline postmaster/forms.py]: https://github.com/lepture/flask-wtf/pull/250

**Pull request merged**